src/test/java/bio/terra/common/CloudPlatformWrapperTest.java

```diff
@@ -0,0 +1,41 @@
+package bio.terra.common;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import bio.terra.common.category.Unit;
+import bio.terra.model.CloudPlatform;
+import java.util.Map;
+import java.util.function.Supplier;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+@Tag(Unit.TAG)
+class CloudPlatformWrapperTest {
+
+  @ParameterizedTest
+  @EnumSource(CloudPlatform.class)
+  void choose_Map(CloudPlatform cloudPlatform) {
+    CloudPlatformWrapper cloudWrapper = (CloudPlatformWrapper.of(cloudPlatform));
+    Map<CloudPlatform, Supplier<String>> cloudMap =
+        Map.of(CloudPlatform.GCP, () -> "GCP", CloudPlatform.AZURE, () -> "Azure");
+    if (cloudWrapper.isGcp()) {
```

Comment:
The test can be made more generic by accepting both the cloud platform and the expected value. For example
```
class CloudPlatformWrapperTest {

  public static Stream<Arguments> platformSource() {
    return Stream.of(
        Arguments.of(CloudPlatform.GCP, "GCP"), Arguments.of(CloudPlatform.AZURE, "Azure"));
  }

  static final Map<CloudPlatform, Supplier<String>> CLOUD_MAP =
      Map.of(CloudPlatform.GCP, () -> "GCP", CloudPlatform.AZURE, () -> "Azure");

  @ParameterizedTest
  @MethodSource("platformSource")
  void choose_Map(CloudPlatform cloudPlatform, String expected) {
    assertEquals(expected, CloudPlatformWrapper.of(cloudPlatform).choose(CLOUD_MAP));
  }

  @ParameterizedTest
  @MethodSource("platformSource")
  void choose(CloudPlatform cloudPlatform, String expected) {
    assertEquals(
        expected, CloudPlatformWrapper.of(cloudPlatform).choose(() -> "GCP", () -> "Azure"));
  }
}
```

Reply:
Oh nice!